### PR TITLE
Cleaned up map.js and removed some features we won't be using like drawing shapes.

### PR DESCRIPTION
**Pull request by kmcostel**
You can no longer draw shapes on the map. The markers on the map will reflect the selected industries and features in the left side menu. Saw no real use for drawing search regions on map, may as well show all markers on map that match the selected features & industries. Next steps for app will be to create the html elements for a menu type of selection on the left side of the page, referencing Mitch's ideas for the design.

### CODE REVIEW

File: src/search.sjs [hunk 1]
```diff
@@ -8,59 +8,54 @@ var input = rawInput.toObject();
...
+}
+else if (input.searchRegions) {
+  //loop through the user's search regions and populate the searchRegions array.
+  for (var i = 0; i < input.searchRegions.features.length; i++) {
+    //decode input from GeoJSON format
+    var r;
+    //xdmp.log(input.searchRegions.features[i]);
+    var geometry = input.searchRegions.features[i].geometry;
+    var coordinates = geometry.coordinates[0];
+    if (coordinates.length === 5 && coordinates[0][0] === coordinates[1][0]
```
Comment (dmcassel) [inline src/search.sjs]: This needs an explanatory comment. What is it doing? Why is that necessary (ie, why do we need to make a box instead of letting it be a polygon)? Communicate with the six-months-from-now developer.

File: data/config/MLFeatures.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{ MarkLogicFeatures: [
+            "Entity Services",
```
Comment (dmcassel) [inline data/config/MLFeatures.json]: Please fix the indentation in this file.

File: data/config/MLFeatures.json [hunk 3]
```diff
@@ -1,9 +1,10 @@
-{ MarkLogicFeatures: [
-            "Entity Services",
-	    "Data Integration",
-            "Data Movement",
-	    "Template Driven Extraction",
-            "SQL"
-	    ]
+{
+  "MarkLogicFeatures": [
```
Comment (kmcostel) [inline data/config/MLFeatures.json]: Still looks like it zig zags a little here, not sure why because it looks fine in sublime...
